pkg/controller/plan/kubevirt.go

```diff
@@ -1389,3 +1397,28 @@ func vmOwnerReference(vm *cnv.VirtualMachine) (ref meta.OwnerReference) {
 	}
 	return
 }
+
+//changes VM name to match DNS1123 RFC convention.
+func changeVmNameDNS1123(currName string, vmID string) string {
+
+	var DNS1123LabelMaxLength int = 63
```

Comment:
i think it will be nicer to remove the `DNS1123` from variable and even from the function. The function description tells what the convention we are based on.